### PR TITLE
add multiple bannertypes

### PR DESCRIPTION
**Pull request by royklip**

### CODE REVIEW

**Comment by martijnjanssen** [comment]
I believe the proportion of the cropped image is not correct, or is it?

**Comment by royklip** [comment]
No, the proportion is not correct and needs to be fixed, unless the other banner type can be scaled or something.

File: src/symposium-app/symposium-committee.html [hunk 1]
```diff
@@ -32,6 +32,7 @@
 
     iron-icon {
       margin-right: 16px;
+      color: orange;
```
Comment (martijnjanssen) [inline src/symposium-app/symposium-committee.html]: Please use the same hex color codes as in the code on other places :smile:

Comment (TimvdLippe) [reply src/symposium-app/symposium-committee.html]: You should probably extract a css-variable for that then.

Comment (martijnjanssen) [reply src/symposium-app/symposium-committee.html]: Yeah, correct.

File: src/symposium-app/symposium-info.html [hunk 2]
```diff
@@ -24,7 +24,7 @@
     </style>
 
     <symposium-page-card>
-      <img src="http://placekitten.com/800/400" alt="The key to your privacy.">
+      <img src="/images/banner.png" alt="The key to your privacy.">
```
Comment (martijnjanssen) [inline src/symposium-app/symposium-info.html]: The `/` at the beginning is not needed.

**Comment by martijnjanssen** [comment]
Could you use the banner without the background? The background color is currently different from the header color. The background of the image could be set to the background color.

File: src/symposium-app/symposium-info.html (hunk 2 above)
Comment (royklip) [reply src/symposium-app/symposium-info.html]: hmm it does not work without the /

Comment (TimvdLippe) [reply src/symposium-app/symposium-info.html]: It should be `../../images/banner.jpg`

File: src/symposium-app/symposium-info.html [hunk 3]
```diff
@@ -12,8 +12,16 @@
...
+    }
+
+    .banner {
+        text-align: center;
+        margin: -16px -16px 0;
+        width: calc(100% + 16px + 16px);
+        background-color: #231657;
     }
 
     @media (max-width: 800px) {
```
Comment (martijnjanssen) [inline src/symposium-app/symposium-info.html]: This is not something you changed, but we can remove the entire media query.

``` css
@media (max-width: 800px) {
    paper-card {
        margin: 0;
    }
}
```

Can be removed.

**Comment by martijnjanssen** [comment]
LGTM.

**Pull request merged**